### PR TITLE
Updated PiiEnabled comments

### PR DESCRIPTION
**Pull request by oldalton**

### CODE REVIEW

File: MSAL/src/public/MSALLogger.h [hunk 1]
```diff
@@ -63,8 +63,7 @@ typedef void (^MSALLogCallback)(MSALLogLevel level, NSString *message, BOOL cont
 @property (readwrite) MSALLogLevel level;
 
 /*!
-    Set to YES to allow messages possibly containing Personally Identifiable Information (PII) to be
-    sent to the logging callback.
+    MSAL provides logging callbacks that assist in diagnostics. There are 2 API callbacks, one for generic messages and the other for messages containing user information. If PiiLoggingEnabled is not set to YES, the returned data for logging will not contain any user information. By default the library will not return any messages with user information in them.
```
Comment (unpluggedk) [inline MSAL/src/public/MSALLogger.h]: We have a single callback, that contains a boolean value to indicate PII or not. I think saying 2 API callbacks might be misleading.
Thus, we should say something in lines of "in a logging callback, there is a boolean value to indicate whether or not the message contains user information. If PiiLoggingEnabled is set to NO, callback will not happen for ..."

Comment (oldalton) [reply MSAL/src/public/MSALLogger.h]: @unpluggedk Does something like this work? I'll run it through review:

`MSAL provides logging callbacks that assist in diagnostics. There is a boolean value in the logging callback that indicates whether the message contains user information.  If PiiLoggingEnabled is set to NO, the returned data for logging will not contain any user information. By default the library will not return any messages with user information in them.`

Comment (unpluggedk) [reply MSAL/src/public/MSALLogger.h]: I am fine with what you wrote though. Just a nit; my preferences.

Comment (oldalton) [reply MSAL/src/public/MSALLogger.h]: updated

**Pull request merged**